django_mongodb/operations.py

```diff
@@ -89,6 +89,13 @@ def convert_decimalfield_value(self, value, expression, connection):
             value = value.to_decimal()
         return value
 
+    def convert_durationfield_value(self, value, expression, connection):
+        if value is not None:
+            # Multiplying value converts milliseconds stored in the database
+            # to microseconds expected by timedelta.
+            value = datetime.timedelta(0, 0, value * 1000)
```

Comment:
If i am not mistaken, timedelta has the following parameters:
class datetime.timedelta(days=0, seconds=0, microseconds=0, milliseconds=0, minutes=0, hours=0, weeks=0)
Maybe we don't need the scalation if we just pass the valle to microseconds.